tl.tl

```diff
@@ -670,6 +670,7 @@ local record tl
    record Token
       x: integer
       y: integer
+      i: integer
```

Comment:
I don't think this value is actually used anywhere, right? I think the `i` in `$EOF$` was a leftover.

Reply:
I'd think it would be nice to expose it to give the original indices into the source, rather than having to reconstruct it from x and y. Perhaps renamed to i_start or something.

Reply:
In the Zig compiler, they actually removed the `x` and `y` entries and only kept the `i` entries, and they derive the line/column info on demand, only when displaying error message (the assumption being that correct code is compiled much more often, so performance should be optimized towards that case: https://youtu.be/IroPQ150F6c?t=2002 ). I briefly considered it, but I think in general it's a lot more useful for us at this stage to have that info easily readable for debugging compared to the minute performance benefit of one fewer integer in a table.

> rather than having to reconstruct it from x and y

Is anybody currently having to reconstruct it? (Not a rhethorical question, I'm just not aware if any other tooling currently needs it.)

Reply:
I'd say let's just leave it out until some time we actually need it, then!